python/caffe/_caffe.cpp

```diff
@@ -51,7 +51,18 @@ const int NPY_DTYPE = NPY_FLOAT32;
 void set_mode_cpu() { Caffe::set_mode(Caffe::CPU); }
 void set_mode_gpu() { Caffe::set_mode(Caffe::GPU); }
 
-void set_random_seed(unsigned int seed) { Caffe::set_random_seed(seed); }
+void InitLog(int level) {
```

Comment:
It'd be nice, but not strictly necessary, to figure out how to do this with boost python call policies instead of hardcoding it. (Like the `Net` constructor.)